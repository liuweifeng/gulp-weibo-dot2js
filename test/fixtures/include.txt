<div>{{include './include' it}}</div>