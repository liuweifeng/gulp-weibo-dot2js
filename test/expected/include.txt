module.exports = function (it) {
var out='<div>';out+=(function(){var tpl = require('./include');return tpl(it);})(); out+='</div>';return out;
}